scripts/releasing.js

```diff
@@ -0,0 +1,100 @@
+const { cat, exec } = require('shelljs');
+
+/**
+ * @param {string[]} rawArgs
+ * @param {string} packageJsonPath
+ * @returns {{ currentVersion: string; newVersion: string }}
+ */
+function getVersion(rawArgs, packageJsonPath) {
+  const args = rawArgs.slice(2).map((arg) => ({
+    value: arg,
+    handled: false,
+  }));
+
+  let IS_HELP = false;
+  let IS_NIGHTLY = false;
+  let IS_FRESH = false;
+  let IS_SET_CUSTOM = false;
```

Comment:
This logic doesn't need to be that complex, we can hardcode it to support only the nightly version.